### DISCUSSION TITLE
Shutter device appears but does not respond

### DISCUSSION THREAD

**Discussion by rucanunes** [make-all/tuya-local#1695] [Q&A]
I have a Tuya Shutter Cover but I guess I'm configuring it wrong. I have the buttons but they don't work.
Do I have to delete the Tuya Integration? I don't have this Shutter switch there...

Localtuya version: v 5.2.1
Home Assistant Core version: core-2024.2.5

[
https://pt.aliexpress.com/item/1005005454334048.html?spm=a2g0o.order_list.order_list_main.34.2b3ccaa4G4b6Z7&gatewayAdapt=glo2bra](url)

<img width="588" alt="309495794-057bc992-6104-4df4-b768-1320250477f1" src="https://github.com/make-all/tuya-local/assets/44728693/a349c5d8-65b8-44a4-a0f0-8e8d4d0e6e21">
<img width="560" alt="309495897-40f0ec59-9fd0-43ca-9a3f-3ab2c9080235" src="https://github.com/make-all/tuya-local/assets/44728693/df612cbe-7534-49fb-95d2-dbf6fa133a41">

**Comment by make-all** [comment]
This is the wrong venue to discuss problems with localtuya
